webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -9634,6 +9633,29 @@ var userOverride = {
                 return '';
             };
 
+            _this.getMessageCreatedAtTime=function(createdAtTime){
+                if(TIME_FORMAT){
+                   var msgtime = new Date(createdAtTime);
+                  
+                   var time = msgtime.toLocaleString('en-US',{
+                       // weekday: 'short', // long, short, narrow
+                       day: 'numeric', // numeric, 2-digit
+                       // year: 'numeric', // numeric, 2-digit
+                        month: 'short', // numeric, 2-digit, long, short, narrow
+                       // hour: 'numeric', // numeric, 2-digit
+                       // minute: 'numeric', // numeric, 2-digit
+                       // second: 'numeric', // numeric, 2-digit
+                    }) + "," + msgtime.getHours() + ":" + msgtime.getMinutes();
+                    // + ":" + msgtime.getSeconds(); 
```

Comment:
![Screenshot 2022-01-05 at 11 29 03 AM](https://user-images.githubusercontent.com/19753380/148167901-81833162-e4c5-4618-8dbb-2bacc1e50680.png)

I guess it should be Jan 5, 05:05.
Also, please remove the commented code if not required.